android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/UriBeaconDiscoveryService.java

```diff
@@ -417,6 +422,11 @@ private void updateNearbyBeaconNotification(boolean single, String url, int noti
         .setContentText(description)
         .setPriority(NOTIFICATION_PRIORITY)
         .setContentIntent(pendingIntent);
+    if (android.os.Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+      if (mPublicUrls.contains(url)) {
+        builder.setVisibility(NOTIFICATION_VISIBILITY);
```

Comment:
Yes. I tried it in two ways:

1. Set `When device is locked` to `Hide sensitive notification content`
2. Turned on `Sensitive` from Apps Notifications Physical Web

In both cases I get `Contents hidden`.